tests/test_client.py

```diff
@@ -2572,3 +2573,10 @@ def test_status_check(client):
 def test_version_check(client):
     res = client.get("/_version")
     assert res.status_code == 200
+
+
+def test_get_dist(client):
+    res = client.get("/_dist")
+    if not res.json:
+        assert res.status_code == 404
+    assert res.status_code == 200
```

Comment:
unit tests should always have the same behavior

ideally we'd want 2 separate test cases:
1. there is a dist config, return it => you can add a dist config to the [test settings](https://github.com/uc-cdis/indexd/blob/master/tests/default_test_settings.py)
2. there is no dist config, return empty list => to test this, you can have a look at how to use a different config than the default. if we can't, only checking case 1 should be okay